content/docs/command-reference/status.md

```diff
@@ -177,24 +163,59 @@ This shows that for stage `bar.dvc`, the dependency `foo` and the
 <abbr>output</abbr> `bar` have changed. Likewise for `foo.dvc`, the dependency
 `foo` has changed, but no output has changed.
 
+## Example: Specific files or directories
+
+`dvc status` only checks the tracked data corresponding to any given `targets`:
+
+```dvc
+$ dvc status foo.dvc dobar
+foo.dvc
+  changed outs:
+          deleted:      foo
+  changed checksum
+dobar
+  changed deps:
+          modified:      bar
+  changed outs:
+          not in cache:      foo
+```
+
+> In this case, the target `foo.dvc` is a `.dvc` file to track the `foo` file,
+> while `dobar` is the name of a stage defined in `dvc.yaml`.
+
+Note that you can check data within directories tracked as a whole, such as the
```

Comment:
I wonder if we can just use "tracked directories"? omit "as a whole"

Reply:
OK, removed phrase "as a whole" completely in 4cac81fa.